src/details/color.rs

```diff
@@ -279,16 +287,21 @@ pub mod color_spaces {
     /// with a [D65 whitepoint.][WhitePoint::D65]
     /// Linear sRGB is equivalent to [BT_709].
     pub const LINEAR_SRGB: ColorSpace = ColorSpace::linear(RGBPrimaries::BT_709, WhitePoint::D65);
```

Comment:
Maybe we should drop LINEAR_ from this now that the convention is ENCODED_ = non-linear?